### DISCUSSION TITLE
Error 6 uploading or uploading and printing from orca to klipper

### DISCUSSION THREAD

**Discussion by chuckkay** [OrcaSlicer/OrcaSlicer#2128] [General]
uploading and printing from orca to klipper printer used to work flawlessly
nothing changed except installing the beta
work perfectly before beta 1.7 beta 1
still works on bambu studio 1.7.4.52

Now the only way to upload and print a model is to download as gcode, go to device tab and upload gcode..

**Comment by tr0tt3rs** [comment]
+1 on this. am also getting the exact same issue. Device tab works fine and connection was tested but unable to upload and print

**Comment by Fennsy** [comment]
I found a way, you have to enter anything as an API key in the printer configuration for version 1.7.0-beta, it's work for me !

https://docs.mainsail.xyz/overview/slicer/cura#direct-g-code-upload

it's for Cura but in Orca it's work

**Comment by tr0tt3rs** [comment]
This still isn't working for me, I'm sure I'm not the only one facing this issue still...

**Comment by m3shary** [comment]
same issue here i can't upload, did you find a fix?

**Comment by m3shary** [comment]
i found a fix that someone mentioned is to type the ip number, so i had to log in to the router and get the ip number then type it in instead of raspberrypi.local 

and it's working now

**Reply by chuckkay** [reply]
Yes Thank you, that worked, sounds like someone messed with something they shouldn’t have, because didn’t need this before, and also don’t need to do this with bambu Studio

**Reply by iammachine84** [reply]
Worked for me .. removed the .local and just replaced with the numerical ip and finally but I tried that a few weeks ago and it didn’t work then but hey it’s working now homer

**Reply by lag0matic** [reply]
+1  Worked.  I updated from 1.7.0 beta to 1.7.0 release and got the bug! At first I thought it was something else, since I'd also done a bunch of cable management on my desk, etc, which involved a few reboots of printer/router/pc! Was chasing ghosts for a bit.

**Reply by Gelisob** [reply]
same, 1.7.0 release. Device tab shows klipper (hostname set to klipperpi.local) and I can do everything, sending print job - error cant resolve host klipperpi.local
adding anything to API box did not change it. Had to move over to using IP address.. first experience with Orca, not very happy! :P
